### ISSUE TITLE
[macOS] UI: No fine-grain control for resolution scale threshold setting

### ISSUE THREAD

**Issue by shinra-electric** [RPCS3/rpcs3#15196]
### Quick summary

The slider for controlling the `resolution scale threshold` setting cannot reach desired values.

### Details

Some games (e.g. Ryu Ga Gotoku series) recommend it to be set to 512x512. 

However, using the slider I can only select 510x510 or 513x513. 

Widening the modal stretches the slider, allowing for a greater range, but 512x512 is still not available. 

### Attach a log file

N/A

### Attach capture files for visual issues

<img width="1512" alt="Screenshot 2024-02-11 at 21 27 48" src="https://github.com/RPCS3/rpcs3/assets/50119606/b2e97757-c89e-44da-85db-6de72f906e1c">

### System configuration

macOS Sonoma 14.3

**Comment by AniLeo** [comment]
Already possible with keyboard arrow keys. Not working on macOS?

**Comment by shinra-electric** [comment]
No, the arrow keys don't seem to do anything.

**Comment by Megamouse** [comment]
on windows you have to move the slider with the mouse before you can use the arrow keys. maybe you did it differently?

**Comment by shinra-electric** [comment]
I don't think so. Using the arrow keys even after using the slider does not change the values. The up/down buttons do scroll the whole view, so perhaps the focus is not being shifted to the slider.

**Comment by Megamouse** [comment]
that sounds like a qt bug.
You should try building a small example and open an issue on their tracker (or maybe there already is one)

**Comment by shinra-electric** [comment]
I found a way to adjust the values on macOS:

- Hover the cursor over the slider control so that it is highlighted
- Swipe up or down using two-finger scrolling to increase/decrease the value one unit at a time

I don't know whether you'd want to close the issue or not though. This method is not intuitive, and I just discovered it by accident...

**Comment by ScottNash042** [comment]
@shinra-electric 
please retest and update

**Comment by shinra-electric** [comment]
No noticeable change. 

- No fine-grain control using mouse pointer
- Arrow keys scroll the window, not the slider
- Fine-grain control possible with the mouse when scrolling up/down when the pointer is hovering over the slider

Even though fine-grain control is possible, it's not obvious to the user. 
A note in the tooltips might help.
